docs/_rpc/ns-eth.md

```diff
@@ -8,6 +8,28 @@ Geth provides several extensions to the standard "eth" JSON-RPC namespace.
 * TOC
 {:toc}
 
+### eth_createAccessList
+
+This method creates an access list based on a given transaction on the current pending state of the blockchain.
+
+The method returns `address` and `storageKeys` and the `gasUsed`.
+
+Example:
+```json
```

Comment:
Your example only shows the output, not the input to `createAccessList`

Reply:
Please see latest commit.